jest-setup.ts

```diff
@@ -36,6 +37,25 @@ Object.defineProperty(
 
 (global as any).__webpack_public_path__ = undefined;
 
+const originalConsoleError = console.error;
+const originalConsoleWarn = console.warn;
+
 beforeAll(() => {
     expect.hasAssertions();
+
+    console.error = (...args: unknown[]) => {
+        if (args.map(String).join().includes('Formik')) {
+            return;
+        }
+
+        originalConsoleError(...args);
+    };
+
+    console.warn = (...args: unknown[]) => {
+        if (args.map(String).join().includes('Formik')) {
+            return;
+        }
+
+        originalConsoleWarn(...args);
+    };
```

Comment:
Hide the warning.
```
console.warn
    Warning: isInitialValid has been deprecated and will be removed in future versions of Formik. Please use initialErrors or validateOnMount instead.
```

Reply:
We may need to replace `formik` or update `isInitialValid` before it is gone.

Reply:
My ideal goal will be to replace `formik` with `react-hook-form` but that will be a project in itself. What does `isInitialValid` do?

Reply:
>What does `isInitialValid` do?

I believe we wanted to bypass form validation with this.
However, in Formik 2, validation always occurs before submission. 
I have removed all `isInitialValid` and ran `e2e` and `test`, and there was no impact.